.travis.yml

```diff
@@ -2,20 +2,22 @@ env:
  # Note: We have to use Cabal 2.4 with GHC 7.0 since for some reason
  # builds of "text" fail with GHC 7.0 and Cabal versions prior to 2.4
  # https://github.com/haskell/bytestring/issues/213
+ # With Cabal 2.4 we also get improved syntax in cabal.project files,
+ # supporting remote sources, and other goodies, so use it across the
+ # board.  This branch is not GHC 7.x compatible, so 8.x-only.
```

Comment:
```suggestion
 # board.
```

Reply:
Thanks, will do.

Reply:
Thanks! This is resolved then, but I don't have the button to mark it… :/